apps/web/components/ui/ImageText/ImageText.vue

```diff
@@ -42,6 +45,9 @@ const getImageUrl = () => {
 
 const getImageDimensions = (): ImageDimensions => {
   switch (viewport.breakpoint.value) {
+    case 'xl': {
+      return { width: 1200, height: 800 };
+    }
```

Comment:
```suggestion
    case '4xl': {
      return { width: 1920, height: 1080 };
    }
```
I assume we shouldn't treat two breakpoints the same here?